openbb_terminal/dashboards/dashboards_controller.py

```diff
@@ -129,13 +138,25 @@ def create_call(other_args: List[str], name: str, filename: str = None) -> None:
         dest="dark",
         help="Whether to show voila in dark mode",
     )
-
+    if streamlit:
```

Comment:
In this controller we use 
```python
 subprocess.Popen(
                f"{cmd} {file} {args}",
                stdout=subprocess.PIPE,
                stderr=subprocess.STDOUT,
                shell=True,
            )
```
which raises a security warning in addition to the fact that this can't be launched from the installer version.

make sure you know how to launch streamlit from the terminal python loop without using the `subprocess` module.
Someone on the team is already working on making the Voila launch from inside the terminal in order to make it available to the users of the installer version

Reply:
if you do `which streamlit` and then `cat` or open that file, you'll see that it's a regular python script that launches some main loop. We can launch the same main loop but from inside the terminal

this is `streamlit` launcher
```
# -*- coding: utf-8 -*-
import re
import sys
from streamlit.web.cli import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```